app/src/utils/storage.ts

```diff
@@ -25,6 +28,28 @@ export const getStorageClientKey = (): ClientKey | null => {
   return null;
 };
 
+export const getExecutorPublicKey = (): Uint8Array | null => {
+  try {
+    if (typeof window !== 'undefined' && window.localStorage) {
+      let clientKeystore: ClientKey = JSON.parse(
```

Comment:
Why do we call it client key? It should be context identity, as client key will become jwt token

Reply:
This will be removed when admin API identity is supported

Reply:
But currently you need client key to talk with server and identity to talk with json rpc

Reply:
ping

Reply:
Will update naming

Reply:
Its not context identity saved in localStorage under context-identity. It is a string as the function takes the first item in the identity array. For now user can't pick which identity to choose